src/main/java/org/jboss/modules/JDKPaths.java

```diff
@@ -55,6 +79,51 @@
     private JDKPaths() {
     }
 
+    private static void processRuntimeImages(final Set<String> jarSet) {
+        try {
+            for (final Path root : FileSystems.getFileSystem(new URI("jrt:/")).getRootDirectories()) {
+                Files.walkFileTree(root, new JrtFileVisitor(jarSet));
```

Comment:
Note that this will pull in all modules, not just the ones that are actually active/visible at run time.  Might or might not be a problem, but we can revisit later.